src/fidesops/ops/service/pagination/pagination_strategy_offset.py

```diff
@@ -53,6 +53,12 @@ def get_next_request(
                 raise FidesopsException(
                     f"Unable to find value for 'limit' with the connector_param reference '{self.limit.connector_param}'"
                 )
+            try:
```

Comment:
I'll pull this out into a separate PR, I just wanted to include this so it could be ready for an early review tomorrow.